Content.Server/Resist/EscapeInventorySystem.cs

```diff
@@ -29,6 +34,20 @@ public override void Initialize()
         SubscribeLocalEvent<CanEscapeInventoryComponent, DroppedEvent>(OnDropped);
     }
 
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+        var query = EntityQueryEnumerator<CanEscapeInventoryComponent>();
+        while (query.MoveNext(out var uid, out var comp))
+        {
+            if ( _timing.CurTime < comp.PenaltyTimer)
+                continue;
+            comp.PenaltyTimer = TimeSpan.MaxValue;
+            EnsureComp<ItemComponent>(uid);
```

Comment:
You should not be removing the ItemComponent and add it later again. If you do that all the information stored inside it gets deleted. Instead you should subscribe to PickupAttemptEvent and cancel it if the penalty time is not over yet.

Reply:
Gotcha. But `PickupAttemptEvent` is for the mob picking up the entity in question, not the entity with the EscapeInventory comp itself. Is `GettingPickedUpAttemptEvent` better?

Reply:
Good catch. Yeah, that one sounds right

Reply:
of course the downside with that (much better) approach is that this isnt predicted on the client but its also REALLY funny to see a lizard get japed trying to kidnap a mouse

Reply:
should be resolved.